src/Allegro.CosmosDb.BatchUtilities/CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres
 to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.0.0] - 2022-09-01
```

Comment:
Version does not match with csproj

Reply:
Is higher for my testing, before completing, i will set 2.0.0